addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -434,6 +439,12 @@ options.Class.include({
     async customizeWebsiteAssets(previewMode, widgetValue, params) {
         await this._customizeWebsite(previewMode, widgetValue, params, 'assets');
     },
+    /**
+     * @see this.selectClass for parameters
+     */
+    async customizeWebsiteColorOrGradient(previewMode, widgetValue, params) {
```

Comment:
I don't see this being used, in this commit at least? :thinking: I may be missing it? :grimacing:

Reply:
It is called through `data-customize-website-color-or-gradient` in the theme background color option.